test/onnx/test_pytorch_onnx_no_runtime.py

```diff
@@ -4,20 +4,22 @@
 
 import io
 import unittest
-from typing import Optional, Type
+from typing import Dict, Optional, Type
 
 import onnx
 
 import torch
 from torch import Tensor
-from torch.onnx import symbolic_helper
+from torch.onnx import OperatorExportTypes, symbolic_helper, utils
```

Comment:
I would avoid importing a class or a function from a module. (https://github.com/pytorch/pytorch/wiki/torch.onnx-Namespacing)

Import only the module instead to keep the name space clean and provide readers with more context on where its members come from. For example, readers don’t need to backtrack to see `sleep` is a function from `time`, as opposed to a function defined in the file. https://google.github.io/styleguide/pyguide.html#22-imports

Reply:
In this case, `import torch._C._onnx as _C_onnx` and use `_C_onnx.OperatorExportTypes`

Reply:
OperatorExportTypes is available in Python, you shouldn't need to access it via `torch._C`.

Reply:
True. I realized this is a test can thus should be considered to be outside of `torch.onnx`